src/common/adapters/jira.js

```diff
@@ -19,7 +19,10 @@ function isJiraPage(loc, doc) {
   return false;
 }
 
-const pathSuffixes = new RegExp('/(browse/[^/]+|projects/[^/]+/issues/[^/]+|secure/RapidBoard.jspa|jira/software/projects/[^/]+/boards/[^/]+/)$', 'g');
+const pathSuffixes = new RegExp(
+  '/(browse/[^/]+|projects/[^/]+/issues/[^/]+|secure/RapidBoard.jspa|jira/software/projects/[^/]+/boards/.*)$',
```

Comment:
`jira/software/projects/[^/]+/boards/.*` captures all possible trailing URL parts of the board view. I opted to not be very specific here and just catch all possible routes within `/boards/` to be more stable against further JIRA changes.

Reply:
Cool. This should also cover what we had here: https://github.com/bitcrowd/tickety-tick/pull/216/files#diff-a4549fad9112b44c628079812d266381L22-R22